App/CrisisCleanup/Sources/Feature/Menu/UI/MenuView.swift

```diff
@@ -7,33 +7,12 @@ struct MenuView: View {
     @ObservedObject var viewModel: MenuViewModel
```

Comment:
Are you moving the selector from the Menu screen? If you open the Android app you can use that as a reference.

Reply:
Reset the changes to this entire file. Nothing in the `MenuView` is change for now.